tex/generic/pgf/utilities/pgfkeys.code.tex

```diff
@@ -180,7 +190,8 @@
 %
 % The length is \pgfkeysvalue{/tikz/length}.
 
-\def\pgfkeysvalueof#1{\csname pgfk@#1\endcsname}
+\let\pgfkeys@relax\relax
+\long\def\pgfkeysvalueof#1{\csname\pgfkeys@ifcsname{pgfk@#1}{pgfk@#1}{pgfkeys@relax}\endcsname}
```

Comment:
I've added an alias for `\relax` for the unlikely event that someone actually does assign to this.